pkgs/development/python-modules/contexttimer/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mock
+}:
+
+buildPythonPackage rec {
+  pname = "contexttimer";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "brouberol";
+    repo = "contexttimer";
+    rev = "v${version}";
+    sha256 = "sha256-pGgJNp+N41c7onDOv9t+iV5JJsXDIFtGMkbyoAyASQE=";
+  };
+
+  patches = [ ./fix.patch ];
```

That name is not very descriptive. 

```suggestion
  patches = [ ./fix-print-and-import.patch ];
```

---

oh that is nicer indeed.